tools/db_stress.cc

```diff
@@ -2363,6 +2367,7 @@ class StressTest {
           FLAGS_universal_max_merge_width;
       options_.compaction_options_universal.max_size_amplification_percent =
           FLAGS_universal_max_size_amplification_percent;
+      options_.atomic_flush = FLAGS_atomic_flush;
```

Comment:
did you set `-disable_wal=true` when testing this?

Reply:
Yes I did.